com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/PollExecuteAppStatusExecution.java

```diff
@@ -28,24 +28,29 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(PollExecuteAppStatusExecution.class);
 
-    enum AppExecutionStatus {
-        EXECUTING(""),
-        SUCCEEDED(""),
-        FAILED("");
+    enum AppExecutionState {
+        EXECUTING, SUCCEEDED, FAILED
+    }
 
-        private String message;
+    private static class AppExecutionStatus {
```

Comment:
Rename to `AppExecutionDetailedState`. `State` and `Status` are synonyms and it's not a good idea to use synonyms to indicate two different things - it's confusing.